backend/api_gateway/group/views.py

```diff
@@ -0,0 +1,52 @@
+import json 
+
+from django.http \
+    import HttpResponseNotAllowed, JsonResponse
+
+from backend.common.command.group_create_command \ 
+    import GroupCreateCommand,  GROUP_CREATE_COMMAND
+from backend.common.command.group_update_command \
+    import GroupUpdateCommand, GROUP_UPDATE_COMMAND
+from backend.common.rpc.infra.adapter.redis.redis_rpc_client \ 
+    import RedisRpcClient
+
+"""
+TODO: add exception controller
+"""
+
+def with_json_response(status, data):
+    return JsonResponse(data=json.dumps(data), status=status, safe=False)
+
+def create_group(request):
+    if request.method == 'POST':
+        return __create_group(request)
+    else:
+        return HttpResponseNotAllowed(['POST'])
+
+def __create_group(request):
+    body = json.loads(request.body.decode())
+    # TODO: check KeyError
+    command = GroupCreateCommand(
+        riders=body['riders'], from_location=body['from_location'], 
+        to_location=body['to_location'])
+
+    result = RedisRpcClient().call(GROUP_CREATE_COMMAND, command)
+
+    # TODO: handling exception
+    return with_json_response(status=204, data=result)
+
+def update_group(request):
+    if request.method == 'PUT':
+        return __update_group(request)
+    else:
+        return HttpResponseNotAllowed(['PUT'])
+
+def __update_group(request):
+    body = json.loads(request.body.decode())
+    # TODO: check KeyError
+    command = GroupUpdateCommand(driver=body['driver'])
+
+    result = RedisRpcClient().call(GROUP_UPDATE_COMMAND, command)
+
+    # TODO: handling exception
+    return with_json_response(status=204, data=result)
```

Comment:
driver -> driver_id

refer to api_gateway/carpool_request/views.py
we need to prepross the result of redisrpcclient.call.